specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transfers.json

```diff
@@ -0,0 +1,1194 @@
+        "operationId": "Transfers_Get",
+        "x-ms-examples": {
+          "TransferGet": {
+            "$ref": "./examples/GetTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/billingProfileName"
+          },
+          {
+            "$ref": "./types.json#/parameters/invoiceSectionName"
+          },
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of transfer.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Transfers"
+        ],
+        "description": "Sends a request to a user in another billing account to transfer billing ownership of their subscriptions. The operation is supported only for billing accounts with agreement type Microsoft Customer Agreement.",
+        "operationId": "Transfers_Initiate",
+        "x-ms-examples": {
+          "InitiateTransfer": {
+            "$ref": "./examples/InitiateTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/billingProfileName"
+          },
+          {
+            "$ref": "./types.json#/parameters/invoiceSectionName"
+          },
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InitiateTransferRequest"
+            },
+            "description": "Request parameters that are provided to the initiate transfer operation."
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Initiated transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "201": {
+            "description": "Initiated transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
```

Comment:
Please be sure that this version of the API returns the `ArmErrorResponse` and not an `ArmError`. BillingRP is making this change going forward.